serverless/containers/index.mdx

```diff
@@ -2,7 +2,93 @@
 meta:
   title: Serverless Containers Documentation
   description: Serverless Containers Documentation
-content:
-  h1: Containers Documentation
-  paragraph: Containers Documentation
 ---
+
+<ProductHeader 
+   productName="Serverless Containers"
+   productLogo="containersWire"
```

Comment:
The image isn't showing up in this and other productHeaders for me - I think you need to take out the "Wire" part from (so it's just "containers") - to check with a local build/develop